lib/model.js

```diff
@@ -2623,7 +2606,34 @@ class Model {
     return this.name;
   }
 
+  /**
+   * Builds a new model instance.
+   * @param {Object}  [values={}] an object of key value pairs
+   * @param {Object}  [options]
+   * @param {Boolean} [options.raw=false] If set to true, values will ignore field and virtual setters.
+   * @param {Boolean} [options.isNewRecord=true]
+   * @param {Array}   [options.include] an array of include options - Used to build prefetched/included model instances. See `set`
+   */
   constructor(values, options) {
+    values = values || {};
+    options = _.extend({
+      isNewRecord: true,
+      $schema: this.$schema,
+      $schemaDelimiter: this.$schemaDelimiter
```

Comment:
Silly me, I just moved the code, commited and opened PR without thinking. You are right of course.